pkg/collector/json_path_collector.go

```diff
@@ -17,10 +18,11 @@ import (
 // querying the pods metrics endpoint and lookup the metric value as defined by
 // the json path query.
 type JSONPathMetricsGetter struct {
-	jsonPath *jsonpath.Compiled
-	scheme   string
-	path     string
-	port     int
+	jsonPath    *jsonpath.Compiled
+	scheme      string
+	path        string
+	port        int
+	reducerFunc string
```

Comment:
In other collectors we call `reducerFunc` for `aggregators` (terminology used in KairosDB). Could we use the same terminology here to be consistent? :)

Reply:
Just pushed a change to rename `reducerFunc` to `aggregator` for consistency purposes!